src/laser/Laser.H

```diff
@@ -14,12 +14,16 @@
 #include <AMReX_RealVect.H>
 #include "utils/Constants.H"
 #include "utils/Parser.H"
+#include <AMReX_MultiFab.H>
 
 class Laser
 {
 public:
-
-    Laser (std::string name);
+/** \brief Read in a laser from an openPMD file */
```

Comment:
Could you document the argument `laser_geom_3D`? You'll find plenty of examples in the rest of the code how to do that.

Reply:
Thanks for the comment. The argument is documented in the file